threaded_io_test.cc

```diff
@@ -27,36 +29,35 @@ namespace {
     }
     return std::pair(sArg, std::string());
   }
-
-  bool checkForIMT(const char* iArg) {
-    std::string sArg(iArg);
-    auto found = sArg.find('/');
-    if(found != std::string::npos) {
-      if(sArg.substr(found+1) != "useIMT") {
-	std::cout <<"unknown option for threads "<<sArg.substr(found+1)<<std::endl;
-	abort();
-      }
-      return true;
-    }
-    return false;
-  }
 }
 
 int main(int argc, char* argv[]) {
   using namespace cce::tf;
 
-  if(not (argc > 1 and argc < 8) ) {
-    std::cout <<"1 to 6 arguments required\n"
-                "threaded_io_test <Source configuration> [# threads[/useIMT]] [# conconcurrent events] [wait time scale factor] [max # events] [<Outputer configuration>]\n";
-    return 1;
-  }
+  CLI::App app{"test different I/O systems under threading"};
 
+  std::string sourceConfig;
+  app.add_option("-s,--source",sourceConfig,"configure Source")->required();
```

Comment:
Do you think at some point having the ability to deal with multiple input files will make sense for this work? For MPI work, I assumed that we will be working with multiple files. If we have the same here, that way the interface can be consistent.